.github/workflows/build_dev_documentation.yml

```diff
@@ -0,0 +1,108 @@
+name: Build dev documentation
+
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build_and_package:
+    runs-on: ubuntu-latest
+    container:
+      image: huggingface/transformers-doc-builder # Not sure if we can also use this image for optimum
```

Comment:
@mishig25 is it fine to use this Docker image for a non-transformers repo like `optimum`?

Reply:
Yes, I think so. I don't think the image contains anything specific to transformers. @LysandreJik might be able to confirm

Reply:
Cool! Do you know what's needed to activate the HuggingFaceDocBuilder bot so one can view the generated HTML from the PR?

Reply:
The `huggingface/transformers-doc-builder` image is built from the following Dockerfile: https://github.com/huggingface/transformers/blob/master/docker/transformers-doc-builder/Dockerfile

It's basically a dev install of `transformers`

Reply:
I have added the necessary secret for the `HuggingFaceDocBuilder` bot to comment on PRs!

Reply:
Thank you very much @LysandreJik 🤗 !